SMPost/examples/example.json

```diff
@@ -6,9 +6,9 @@
   "postId": "21098319",
   "hasLanguage": "en",
   "platform": "Twitter",
-  "hasText": "This is a tweet",
-  "hasImage": "https://twt.com/image.png",
-  "hasVideo": "https://twt.com/video.mp4",
+  "hasText": ["This is a tweet", "This is another tweet"],
+  "hasImages": ["https://twt.com/image.png", "https://twt.com/image2.png", "https://twt.com/image3.png"],
+  "hasVideos": ["https://twt.com/video.mp4", "https://twt.com/video2.mp4"],
   "hasPrivacy": "public",
   "location": {
     "type": "Point",

```

Comment:
this is not a valid point. coordinates required.

Reply:
I don't understand what is wrong here. Coordinates are right below "type": "Point".

Reply:
I do not know either, maybe I misplaced the comment. right.